### ISSUE TITLE
IOException on server reload

### ISSUE THREAD

**Issue by ScarletBlizzard** [TBMCPlugins/CustomDimensions#3]
```
[22:43:08 WARN]: java.io.IOException: The process cannot access the file because it is being used by another process
[22:43:08 WARN]:        at sun.nio.ch.FileDispatcherImpl.write0(Native Method)
[22:43:08 WARN]:        at sun.nio.ch.FileDispatcherImpl.write(Unknown Source)
[22:43:08 WARN]:        at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
[22:43:08 WARN]:        at sun.nio.ch.IOUtil.write(Unknown Source)
[22:43:08 WARN]:        at sun.nio.ch.FileChannelImpl.write(Unknown Source)
[22:43:08 WARN]:        at net.minecraft.server.v1_16_R2.SessionLock.a(SourceFile:40)
[22:43:08 WARN]:        at net.minecraft.server.v1_16_R2.Convertable$ConversionSession.<init>(Convertable.java:218)
[22:43:08 WARN]:        at buttondevteam.customdimensions.CustomDimensions$1.<init>(CustomDimensions.java:57)
[22:43:08 WARN]:        at buttondevteam.customdimensions.CustomDimensions.load(CustomDimensions.java:57)
[22:43:08 WARN]:        at buttondevteam.customdimensions.CustomDimensions.onEnable(CustomDimensions.java:24)
[22:43:08 WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[22:43:08 WARN]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380)
[22:43:08 WARN]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483)
[22:43:08 WARN]:        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugin(CraftServer.java:499)
[22:43:08 WARN]:        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugins(CraftServer.java:413)
[22:43:08 WARN]:        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.reload(CraftServer.java:938)
[22:43:08 WARN]:        at org.bukkit.Bukkit.reload(Bukkit.java:699)
[22:43:08 WARN]:        at org.bukkit.command.defaults.ReloadCommand.execute(ReloadCommand.java:54)
[22:43:08 WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
[22:43:08 WARN]:        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.dispatchCommand(CraftServer.java:800)
[22:43:08 WARN]:        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.dispatchServerCommand(CraftServer.java:762)
[22:43:08 WARN]:        at net.minecraft.server.v1_16_R2.DedicatedServer.handleCommandQueue(DedicatedServer.java:409)
[22:43:08 WARN]:        at net.minecraft.server.v1_16_R2.DedicatedServer.b(DedicatedServer.java:376)
[22:43:08 WARN]:        at net.minecraft.server.v1_16_R2.MinecraftServer.a(MinecraftServer.java:1211)
[22:43:08 WARN]:        at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:999)
[22:43:08 WARN]:        at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177)
[22:43:08 WARN]:        at java.lang.Thread.run(Unknown Source)
```

Getting IOException on server reload.

Server: Paper 1.16.3 #192
System: Windows 10

**Comment by alisolarflare** [comment]
Are you using minecraft:reload, spigot's /reload command, or manually shutting down and restarting the server?

**Comment by ScarletBlizzard** [comment]
I'm using Spigot's command.

**Comment by NorbiPeti** [comment]
Fixed in 1.3.
